App.Loans/Controllers/LoansController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using App.Loans.Interface;
+
+namespace App.Loans.Controllers
+{
+
+        [Route("api/loans")]
+        [ApiController]
+        public class LoansController : ControllerBase
+        {
+            readonly IAnotherService _anotherService;
+            readonly ILoanManger _loansManager;
+            public LoansController(
+                IAnotherService anotherService,
+                ILoanManger loansManager)
+            {
+                _anotherService = anotherService;
+                _loansManager = loansManager;
+            }
+
+            // GET api/loans/values
+            [HttpGet("values")]
+            public ActionResult<IEnumerable<string>> GetActiveLoans()
```

Comment:
Endpoint, according to the name, should return a list of loans, but it returns a list of strings. Why ?